src/lib/Form/Type/Language/LanguageChoiceType.php

```diff
@@ -24,12 +25,16 @@ class LanguageChoiceType extends AbstractType
     /** @var LanguageService */
     protected $languageService;
 
+    /** @var ConfigResolver */
+    protected $configResolver;
+
     /**
      * @param LanguageService $languageService
      */
-    public function __construct(LanguageService $languageService)
+    public function __construct(LanguageService $languageService, ConfigResolver $configResolver)
     {
         $this->languageService = $languageService;
+        $this->configResolver = $configResolver;
```

Comment:
No need for `ConfigResolver` here, why don't just inject `array $siteAccessLanguages`?

```yml
EzSystems\EzPlatformAdminUi\Form\Type\Language\LanguageChoiceType:
    arguments:
        $siteAccessLanguages: '$languages$'
```

Reply:
I can, but as service was autowired I did not want to having to define it.

Also question is if we allow siteaccess switching at some point (several admins), then it might be worthwhile to get it dynamically via ConfigResolver on-demand.

But that said, let's define it for now, can always come back and change it if we need to.